tests/drivers/can/api/board/mimx93_evk_a55.overlay

```diff
@@ -0,0 +1,28 @@
+/{
+    chosen{
+        zephyr,canbus = &flexcan2;
+    };
+};
+
+&lpi2c2 {
+    status = "okay";
+};
+
+&mfd0 {
+    status = "okay";
+};
+
+&gpiohog_exp_sel {
+    status = "okay";
+    output-low;
+};
```

Comment:
How do this change relate to CAN?

Reply:
this gpio controls a series of muxes on the board. pull it low yo select can signal.